src/providers/esteem/esteem.js

```diff
@@ -71,6 +71,54 @@ export const updateDraft = data => new Promise((resolve, reject) => {
     });
 });
 
+export const addBookmark = (username, author, permlink) => api
+  .post('/bookmark', {
+    username,
+    author,
+    permlink,
+    chain: 'steem',
+  })
+  .then(resp => resp.data);
+
+/**
+ * @params current username
+ */
+export const getBookmarks = username => api.get(`/bookmarks/${username}`).then(resp => resp.data);
+
+/**
+ * @params id
+ * @params current username
+ */
+export const removeBookmark = (id, username) => api.delete(`/bookmarks/${username}/${id}`);
+
+/**
+ * @params current username
+ */
+export const getFavorites = username => api.get(`/favorites/${username}`).then(resp => resp.data);
+
+/**
+ * @params current username
+ * @params target username
+ */
+export const isFavorite = (currentUsername, targetUsername) => api.get(`/isfavorite/${currentUsername}/${targetUsername}`).then(resp => resp.data);
```

Comment:
I think this line is long for eslint rules

Reply:
nope its not getting error